controllers/login.js

```diff
@@ -0,0 +1,26 @@
+const ModelUser = require('../models/user')
+const Guid = require('guid')
+
+module.exports.login = async (req, res) => {
```

Comment:
i am going to prefer to do not use an arrow function, is better if you assign:
`function login(){}` because if you get an error you will be unable to track the error

Reply:
Done, thanks!